Library/Formula/snakemake.rb

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
```

Comment:
That's courtesy of [homebrew-pypi-poet](https://pypi.python.org/pypi/homebrew-pypi-poet).

Should I drop it?